src/java.desktop/unix/native/libsplashscreen/splashscreen_sys.c

```diff
@@ -372,7 +372,9 @@ void
 sendctl(Splash * splash, char code) {
 //    if (splash->isVisible>0) {
     if (splash && splash->controlpipe[1]) {
-        write(splash->controlpipe[1], &code, 1);
+        if (write(splash->controlpipe[1], &code, 1) != 1) {
+            fprintf(stderr, "Cannot not write to splash screen control: %s\n", strerror(errno));
+        }
```

Comment:
what happens to the process if this call fails ?
Can it continue ?
You can check by commenting out the call and see what happens when you run an app on Linux that uses splashScreen ? SwingSet2 would be sufficient.
And assuming it is a continuable error, can you do the print only in a DEBUG build ?